plugins/vpp/l3plugin/vppcalls/vpp2005/vrrp_dump.go

```diff
@@ -0,0 +1,93 @@
+//  Copyright (c) 2020 Cisco and/or its affiliates.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at:
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package vpp2005
+
+import (
+	"net"
+
+	"go.ligato.io/vpp-agent/v3/plugins/vpp/binapi/vpp2005/ip_types"
+	"go.ligato.io/vpp-agent/v3/plugins/vpp/binapi/vpp2005/vrrp"
+	l3 "go.ligato.io/vpp-agent/v3/proto/ligato/vpp/l3"
+)
+
+// DumpVrrpEntries dumps all configured VRRP entries.
+func (h *VrrpVppHandler) DumpVrrpEntries() (entries []*l3.VRRPEntry, err error) {
+	req := &vrrp.VrrpVrDump{
+		SwIfIndex: 0xffffffff, // Send multirequest to get all VRRP entries
+	}
+	reqCtx := h.callsChannel.SendMultiRequest(req)
+	for {
+		vrrpDetails := &vrrp.VrrpVrDetails{}
+		stop, err := reqCtx.ReceiveReply(vrrpDetails)
+		if stop {
+			break
+		}
+		if err != nil {
+			return nil, err
+		}
+
+		// VRRP interface
+		ifName, _, exists := h.ifIndexes.LookupBySwIfIndex(uint32(vrrpDetails.Config.SwIfIndex))
+		if !exists {
+			h.log.Warnf("VRRP dump: interface name not found for index %d", vrrpDetails.Config.SwIfIndex)
```

Comment:
This will store `Interface` field as empty even though it refers to some unknown interface.. not sure what would be the right way to process such data.. @milanlenco  thoughts?

Reply:
I think this is OK because as a result the VRRP entry will be determined as not being in-sync and Update/Re-create should then take care of that.